collections/rh-php72-rh/check-version/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+source ${THISDIR}/../../../common/functions.sh
+source ${THISDIR}/../include.sh
+
+out=$(scl enable $ENABLE_SCLS "php -v")
+ret=$?
+
+echo "$out"|grep -o 'PHP 7\.1\.'
```

Comment:
I think that should be `7\.2\.`:

```suggestion
echo "$out"|grep -o 'PHP 7\.2\.'
```

Reply:
indeed; fixed (in a simpler way which will work for 7.3, 7.4, ...)